securedrop/encryption.py

```diff
@@ -34,27 +36,14 @@ class GpgDecryptError(Exception):
 
 class EncryptionManager:
 
-    GPG_KEY_TYPE = "RSA"
-    GPG_KEY_LENGTH = 4096
-
-    # All reply keypairs will be "created" on the same day SecureDrop (then
-    # Strongbox) was publicly released for the first time.
-    # https://www.newyorker.com/news/news-desk/strongbox-and-aaron-swartz
-    DEFAULT_KEY_CREATION_DATE = date(2013, 5, 14)
-
-    # '0' is the magic value that tells GPG's batch key generation not
-    # to set an expiration date.
-    DEFAULT_KEY_EXPIRATION_DATE = "0"
-
     REDIS_FINGERPRINT_HASH = "sd/crypto-util/fingerprints"
     REDIS_KEY_HASH = "sd/crypto-util/keys"
 
-    SOURCE_KEY_NAME = "Source Key"
     SOURCE_KEY_UID_RE = re.compile(r"(Source|Autogenerated) Key <[-A-Za-z0-9+/=_]+>")
 
-    def __init__(self, gpg_key_dir: Path, journalist_key_fingerprint: str) -> None:
+    def __init__(self, gpg_key_dir: Path, journalist_pub_key: Path) -> None:
         self._gpg_key_dir = gpg_key_dir
-        self._journalist_key_fingerprint = journalist_key_fingerprint
+        self.journalist_pub_key = journalist_pub_key
```

Comment:
I think this is good 👍

Reply:
Optionally, you could check if the file exists here.

Reply:
Oh good point, done.